docker-compose.yml

```diff
@@ -60,5 +61,16 @@ services:
             - postgres
         volumes:
                 - ./airflow:/usr/local/airflow
+                - ./meltano:/usr/local/meltano
```

Comment:
It's great that we have a meltano mount, but because the `.meltano` is in our `.gitignore`, we have to make sure that when `docker compose up` runs, the container has access to all of necessary extractors and loaders, which live inside `.meltano`.

I would suggest in the `command` section, after we run `python3 demo-setup/datawarehouse_setup.py`, that we `cd` into the meltano project folder and run `meltano install`, to ensure `.meltano` is available in the meltano project folder.

Reply:
lgtm